### DISCUSSION TITLE
Resolving source/destination.ip

### DISCUSSION THREAD

**Discussion by mjakubowskilkr** [Security-Onion-Solutions/securityonion#7132] [General] [answered]
Is it possible to resolving local IPs from revDNS? How to set it up?

**Comment by dougburks** [comment]
This question could have a few different meanings. Please clarify exactly what you're trying to do.

**Comment by mjakubowskilkr** [comment]
I mean resolving my local names e.g in Alerts module – currently it shows only ip, I’ve got dns server (AD) so my local IPs are resolvable, I don’t want to resolve public IPs.

**Comment by dougburks** [comment, marked as answer]
If you have an internal web service that can resolve IP addresses to names, then you could add that web service as a custom action:
https://docs.securityonion.net/en/2.3/soc-customization.html#action-menu

Alternatively, you might be able to modify the ingest pipeline to perform the DNS lookups. However, please note that can have serious performance implications for your ingest pipeline.
